component/disassembler/disassembler.js

```diff
@@ -435,4 +435,20 @@ function Disassembler() {
     this.disassemble = disassembleMemory;
     this.loadSymbols = loadSymbols;
     this.labelAddress = labelAddress;
-}
\ No newline at end of file
+}
+
+if (typeof module === 'object' && typeof module.exports === 'object') {
```

Comment:
This is not very clear. I guess that it tests whether you are running in browser or electron/nodejs environment, is it possible to have a (global?) function instead to make it clearer?

Reply:
Deleted.